R/build-favicons.R

```diff
@@ -1,20 +1,13 @@
-#' Create favicons from package logo
+#' @details
 #'
-#' This function auto-detects the location of your package logo (with the name
-#' `logo.svg` (recommended format) or `logo.png`) and runs it through the
-#' <https://realfavicongenerator.net> API to build a complete set of favicons
-#' with different sizes, as needed for modern web usage.
-#'
-#' You only need to run the function once. The favicon set will be stored in
-#' `pkgdown/favicon` and copied by [init_site()] to the relevant location when
+#' You only need to run `build_favicons()` once. The favicon set will be stored in
```

Comment:
I agree that `build_favicons()` is probably better as `init_favicons()` since it does imply that you don't need to run it very often. But I think you're likely to run it even less than `init_site()` (and indeed in most cases pkgdown will run it for you) so I think documenting with `init_site()` makes it maybe a bit too prominent.